spec/controllers/queue_items_controller_spec.rb

```diff
@@ -90,11 +80,122 @@
         expect(QueueItem.count).to eq(1)
       end
 
+      it "normalizes the remaining queue items" do
+        queue_item1 = Fabricate(:queue_item, user: user, order:1)
+        queue_item2 = Fabricate(:queue_item, user: user, order:2)
+        delete :destroy, id: queue_item1.id
+        expect(queue_item2.reload.order).to eq(1)
+      end
+
       it "redirects to my queue path" do
         queue_item = Fabricate(:queue_item, user: user)
         delete :destroy, id: queue_item.id
         expect(response).to redirect_to my_queue_path
       end
     end
+    describe "POST #update_queue" do
+      it "deletes a rating if input is set to blank" do
+      end
+
+      it "creates review if one does not already exist on queue item" do
+        queue_item1 = Fabricate(:queue_item, order:1, user:user)
+        post :update_queue, queue_items: 
+          [{ id: queue_item1.id, 
+            order:1, 
+            video_id: queue_item1.video_id, 
```

Comment:
as I mentioned before, you don't need to pass `video_id` as fetching the queue_item by its primary key id should grant you an easy access to video_id if you need that